src/const.ts

```diff
@@ -155,6 +155,7 @@ export interface Constant extends Required<ConstantProps> {
   botStudioEditProps?: BotStudioEditProps;
   widgetOpenState?: boolean;
   onWidgetOpenStateChange?: (params: OnWidgetOpenStateChangeParams) => void;
+  onWidgetExpandStateChange?: (isExpanded: boolean) => void;
```

Comment:
Could you rebase from the develop branch and mark this one like others?
```
  /**
   * @private Callback to be called when the widget expand state changes.
   */
  onWidgetExpandStateChange?: (isExpanded: boolean) => void;
```

Reply:
nice!

Reply:
done!